sdcm/utils/ldap.py

```diff
@@ -58,6 +58,8 @@ def is_ldap_connection_bound():
     @staticmethod
     def add_ldap_entry(ip, ldap_port, user, password, ldap_entry):
         if not LdapContainerMixin.is_ldap_connection_bound():
+            sleep(5)
+            print('this is creating a ldap_connection with values {}'.format([ip, ldap_port, user, password]))
```

Comment:
Can you switch to logging?

Reply:
Also can you add comment to explain this sleep ?

And if important add it also in master, cause soon we'll branch 2021.1...

Reply:
switched, @amoskong .
@fruch , on previous tries I didn't see the need for it, but as this one was failing, I added as debug, and altogether with other changes, it won't bother, and if we find that we want to add this further, i will then send a PR to master and branch-2021.1